SublimeSkinz/Prebid.js
# DL-1862: moving notifyId from an external to an internal ref

## Type of change


## Description of change

- setting our notifyId as an internal ref instead of a bid param;
- fixing device detection regexps.

## How to test

- execute:

   ```bash
  gulp serve --modules=sublimeBidAdapter  --file test/spec/modules/sublimeBidAdapter_spec.js
   ```

This will lint, test, compile prebid.js with the sublimeBidAdapter and launch a test server;

- check that tests are ✅
- [in single-notifyId.html](https://github.com/SublimeSkinz/Emilbus/blob/master/sac/resources/test-display/prebidjs/single-notifyId.html):

  - L51-55, remove the notifyId code;

  - L66, change:

```js
    <script src="prebid-certified.js"></script>
```

...to:

```js
    <script src="http://localhost:9999/build/dev/prebid.js"></script>
```

- uncomment an ad in smartRequest.js

You _should_ see it on http://sac.ayads.co.local/resources/test-display/prebidjs/single-notifyId.html

## Other information

https://sublimeskinz.atlassian.net/browse/DL-1862
